clients/drcachesim/reader/compressed_file_reader.cpp

```diff
@@ -52,7 +52,13 @@ namespace drmemtrace {
 bool
 open_single_file_common(const std::string &path, gzFile &out)
 {
-    out = gzopen(path.c_str(), "rb");
+    if (path == "-") {
+        // We assume stdin is 0.
```

Comment:
Curious: is there any env where stdin is not zero?

Reply:
A process can dup stdin==0 to a different number and close 0.